homeassistant/components/template/light.py

```diff
@@ -378,6 +378,9 @@ def update_brightness(self):
             return
         try:
             brightness = self._level_template.async_render()
+            if brightness is None or brightness == "None" or brightness == "":
```

Comment:
template rendering always returns a string.

Reply:
Fair enough. Removed the raw `None`